### PR TITLE
docs: Add missing options to man page and CLI --help info

### LINKED ISSUE #488: bug: Undocumented options
- The version of eza being used (`eza --version`)

v0.14.0 [+git]

- The command-line arguments you are using

`-?`

- Your operating system and hardware platform

Linux Mint 21.2

---

The following options are not documented in the man file:

`-?`, `--help`
`-v`, `--version`

The following options are not documented in either the man file or the `--help` output:

`-A`, `--almost-all`
`-X`, `--dereference`

### PR DESCRIPTION
**Pull request by cgorski**
Add missing --dereference, --almost-all, --help, --version to man pages. 
Add --dereference, --almost-all to CLI --help message.

Fixes #488

### CODE REVIEW

File: man/eza.1.md [hunk 1]
```diff
@@ -38,6 +38,16 @@ EXAMPLES
 : Displays a tree of files, three levels deep, as well as each file’s metadata.
 
 
+META OPTIONS
+===============
+
+`-?`, `--help`
```
Comment (cafkafk) [inline man/eza.1.md:44]: We wanna move away from `-?` and to `-h` instead by `v1.0`, maybe we should avoid advertising it.
```suggestion
`--help`
```

**Review by cafkafk — CHANGES_REQUESTED** [review]
Maybe while here you could remove `-h` as well, as we will be removing it in the future?

**Review by MartinFillon — COMMENTED** [review]
This is great but could you add those modifications to the different completions if not already done ?

**Comment by cgorski** [comment]
> This is great but could you add those modifications to the different completions if not already done ?

Can you rephrase that? I’m not sure what you mean.

**Comment by cgorski** [comment]
@MartinFillon ah - just looked through the code and I'm assuming you meant bash completions. Taking a look.

**Comment by MartinFillon** [comment]
@cgorski nah all completions not only bash but zsh fish

**Review by PThorpe92 — APPROVED** [review]

**Review by cafkafk — APPROVED** [review]
Thanks! LGTM, merge incoming
